lib/events/complete_test.go

```diff
@@ -248,3 +257,29 @@ func (m *mockSessionTrackerService) RemoveSessionTracker(ctx context.Context, se
 func (m *mockSessionTrackerService) UpdatePresence(ctx context.Context, sessionID, user string) error {
 	return trace.NotImplemented("UpdatePresence is not implemented")
 }
+
+type mockSessionTrackerServiceAccessDenied struct{}
+
+func (m *mockSessionTrackerServiceAccessDenied) GetActiveSessionTrackers(ctx context.Context) ([]types.SessionTracker, error) {
+	return nil, trace.AccessDenied("Access denied")
```

Comment:
super nit: uppercase error string

```suggestion
	return nil, trace.AccessDenied("access denied")
```

Same for others.